### PR TITLE
Rcc: finish documenting stm32f4 and stm32f7

### PR DESCRIPTION
**Pull request by aurelj**
This patch set adds the missing registers/fields and fix the incorrect ones, and documents all the fields for the Rcc device of every stm32f4 and stm32f7.

### CODE REVIEW

File: devices/stm32f413.yaml [hunk 1]
```diff
@@ -62,6 +62,45 @@ RCC:
...
+      - LPTIMER1SEL
+    _add:
+      PLLI2SDIVR:
+        description: PLLI2S division factor for SAI1 A/B clock
+        bitOffset: 0
+        bitWidth: 5
+      PLLDIVR:
+        description: PLL division factor for SAI1 A/B clock
+        bitOffset: 8
+        bitWidth: 5
```
[the remark this answers is not in the document]
Comment (aurelj) [reply devices/stm32f413.yaml]: I saw this as well. I have no way to check for sure which one is right, but 12:8 is matching the other stm32f4 devices so this makes more sense to me.

File: peripherals/rcc/rcc_v2_dckcfgr2_cecsel.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+RCC:
+  DCKCFGR2:
+    CECSEL:
+      LSE: [0, "LSE clock is selected as HDMI-CEC clock"]
+      HSI: [1, "HSI divided by 488 clock is selected as HDMI-CEC clock"]
```
[the remark this answers is not in the document]
Comment (aurelj) [reply peripherals/rcc/rcc_v2_dckcfgr2_cecsel.yaml]: Right, that is more explicit.

File: devices/stm32f413.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamgreig) [reply devices/stm32f413.yaml]: Yea ok, matching the others is good enough!

File: peripherals/rcc/rcc_v2_dckcfgr_dfsdm_f4.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+RCC:
+  DCKCFGR:
+    CKDFSDM?ASEL:
+      I2S1: [0, "CK_I2S_APB1 selected as audio clock"]
+      I2S2: [1, "CK_I2S_APB2 selected as audio clock"]
```
[the remark this answers is not in the document]
Comment (aurelj) [reply peripherals/rcc/rcc_v2_dckcfgr_dfsdm_f4.yaml]: That's what I wrote at first, but looking at the Clock Tree diagram in the RM, this bit is obviously selecting between the I2S1 clock and I2S2 clock, so I think this make more sense this way.

File: peripherals/rcc/rcc_v2_dckcfgr_dfsdm_f4.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+    CKDFSDM1SEL:
+      APB2: [0, "APB2 clock USED as Kernel clock"]
```
[the remark this answers is not in the document]
Comment (aurelj) [reply peripherals/rcc/rcc_v2_dckcfgr_dfsdm_f4.yaml]: OK.

File: peripherals/rcc/rcc_v2_dckcfgr_dfsdm_f4.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamgreig) [reply peripherals/rcc/rcc_v2_dckcfgr_dfsdm_f4.yaml]: Okay, makes sense.

**Comment by aurelj** [comment]
Indeed, this PR took me quite a long time. Thanks for the detailed review ! I know this can be daunting as well to do such a review, but you've showed once again that this is always useful.
PR updated.

**Comment by adamgreig** [comment]
Great, thanks! It's a credit that in 850 lines changed there's only one error! I'll merge once Travis passes.

**Pull request merged**